AnnotationModule.py

```diff
@@ -99,4 +120,35 @@ def xmlMask(s,params):
         s["warnings"].append(
             f"After AnnotationModule.xmlMask NO tissue remains detectable! Downstream modules likely to be incorrect/fail")
 
+    return
+
+def geoJSONMask(s, params):
+    logging.info(f"{s['filename']} - \tgeoJSONMask")
+    mask = s["img_mask_use"]
+
+    geojson_basepath = params.get("geojson_filepath",None)
+    geojson_suffix = params.get("geojson_suffix", "")
+    if not geojson_basepath:
+        geojson_basepath = s["dir"]
+
+    fname = geojson_basepath + os.sep + PurePosixPath(s['filename']).stem + geojson_suffix + '.json'

```

Comment:
The configuration reads: "leave blank to replacing image filename extension with "geojson""

But here the image extension it looks for is 'json'.

Reply:
Good find! Will fix.